routing/turns_sound.hpp

```diff
@@ -56,6 +56,11 @@ class TurnsSound
   /// After the second notification has been pronounced
   /// m_nextTurnNotificationProgress == Second.
   PronouncedNotification m_nextTurnNotificationProgress;
+  /// The flag is set to true if notification about the second turn was pronounced.
```

Comment:
By the way, according to coding standard private section follows after public and protected.